cfgov/unprocessed/css/organisms/video-player.less

```
@@ -48,18 +53,52 @@
 
 .o-video-player_image-container {
     position: relative;
-    // @black not used because it's not totally black.
-    background-color: #000;
+    background-color: #fff;
     text-align: center;
     height: 100%;
-    width: 100%;
+    display: flex;
+
+    // Create an aspect ratio box, see https://css-tricks.com/aspect-ratio-boxes/#the-psuedo-element-tactic
+    &:before {
+        content: '';
+        width: 1px;
+        margin-left: -1px;
+        float: left;
+        height: 0;
+        // 1280px / 720px (16/9) aspect ratio.
+        padding-top: 720px / 1280px * 100%;
+    }
+
+    // To clear float.
+    &:after {
+        content: '';
+        display: table;
+        clear: both;
+    }
 }
 
 .o-video-player_image {
     // Remove space below image created by inline display spacing.
     display: block;
     opacity: 0;
     transition: opacity .25s;
+    // Needed for IE11 to not stretch the image.
+    flex-shrink: 0;
+
+    // Using the default CFPB cover card image.
+    &[src^="/static/img/cfpb_video_cover_card_954x200"] {
+        padding: 10px;
+        box-sizing: border-box;
+        align-self: center;
+
+        // Overrides values set in featured-content-module.less file in the DS.
+        &.o-featured-content-module_img {
+            max-width: 100%;
+            height: auto;
+            left: 0;
+            transform: none;
+        }
+    }
```

Suggesting increasing the padding when used alone (as on http://localhost:8000/consumer-tools/everyone-has-a-story/leah-repaying-student-loans/) and then overriding to bring it down to something more reasonable when in an FCM (suggesting bumping that to 15px since we use that value more often).

Also, should these values be calculated from `@grid_gutter-width`?

```suggestion
    &[src^="/static/img/cfpb_video_cover_card_954x200"] {
        padding: 60px;
        box-sizing: border-box;
        align-self: center;

        // Overrides values set in featured-content-module.less file in the DS.
        &.o-featured-content-module_img {
            max-width: 100%;
            height: auto;
            padding: 15px;
            left: 0;
            transform: none;
        }
    }
```

---

Updated in https://github.com/cfpb/consumerfinance.gov/pull/6284/commits/14b49c067cee32f11f3bc5a4b5e87bc495a8415e to use ems. 

> Also, should these values be calculated from `@grid_gutter-width`?

Hmmm these seem kind of arbitrary values that fit with the image but happen to be divisible by `@grid_gutter-width`, right? I've left them as hardcoded value (but converted to ems).

---

Yeah, fair point.